### PR TITLE
New design

### PR DESCRIPTION
**Pull request by johan-t**
minor changes to ui

### CODE REVIEW

**Comment by johan-t** [comment]
I will fix the tests later

**Comment by brandesdavid** [comment]
Screenshot 2024-04-20 at 17 21 06
why is the button stretched out xD
Screenshot 2024-04-20 at 17 21 41
does the down button have a lighter color?
Screenshot 2024-04-20 at 17 23 18
could you move the down button a bit more to the right please?

what about text padding?

**Comment by brandesdavid** [comment]
great changes though!!!

File: src/components/Buttons/ReportButton/ReportButton.css [hunk 1]
```diff
@@ -2,22 +2,14 @@
     position: fixed;
     height: 90px;
```
Comment (brandesdavid) [inline src/components/Buttons/ReportButton/ReportButton.css:3]: maybe use min-height and min-width?

**Review by brandesdavid — CHANGES_REQUESTED** [review]
a question: was the figma only for orientation or did you wanted to use an exact replication of the styles?

**Comment by johan-t** [comment]
> a question: was the figma only for orientation or did you wanted to use an exact replication of the styles?

I used it just as a reference as some of it was not applicable or consistent

**Comment by johan-t** [comment]
> Screenshot 2024-04-20 at 17 21 06
> 
> why is the button stretched out xD
> 
I think it looks stretched because it is much bigger than the utility  button, but it is not streched irl
> Screenshot 2024-04-20 at 17 21 41
> 
> does the down button have a lighter color?
> 
Yes I asked you if I could use the same color as the location marker to be consistent 
> Screenshot 2024-04-20 at 17 23 18
> 
> could you move the down button a bit more to the right please?

Yes will do 
> 
> 
> 
> what about text padding?

Standard text padding is sub optimal as this it is very different for each situation meaning we would have to overwrite it every time

**Comment by brandesdavid** [comment]
@johan-t 

The button is stretched. (100x90)
Screenshot 2024-04-21 at 11 02 25

when changing top to -30px and -33px, the down button  ```button.close-button``` border gets lighter. that is what i mean. 
Screenshot 2024-04-21 at 11 09 49

Screenshot 2024-04-21 at 11 09 53

**Comment by johan-t** [comment]
> @johan-t
> 
> The button is stretched. (100x90) Screenshot 2024-04-21 at 11 02 25
> 
> when changing top to -30px and -33px, the down button `button.close-button` border gets lighter. that is what i mean. Screenshot 2024-04-21 at 11 09 49
> 
> Screenshot 2024-04-21 at 11 09 53

you are right , oopsie

**Comment by brandesdavid** [comment]
changed some styles, could you maybe please add some documentation for styling? (like classes or variables etc)
but you can review now and merge if you want to @johan-t

**Comment by johan-t** [comment]
> changed some styles, could you maybe please add some documentation for styling? (like classes or variables etc) but you can review now and merge if you want to @johan-t

What kind of documentation were you thinking? I am not sure if we need docs for the variables as they are written in plain english same for the classes and it would be super long to have documentation for each class. Or are there specific classes that need documentation_
